docs/customize/emails.md

```diff
@@ -43,6 +43,15 @@ worker_1      | 2021-11-16T12:47:41.816520657Z This is my wonderful email-templa
 worker_1      | 2021-11-16T12:47:41.816529772Z -------------------------------------------------------------------------------
 ```
 
+Notification emails are configured separately from the account-related email settings. Notifications are things like 
+the results of the asynchronous task that automatically checks whether files have been corrupted, meaning that a file's checksum differs from its original checksum.
+
+Notification e-mail options can be modified by editing the following configurations:
+
+- `MAIL_DEFAULT_SENDER`: modifies the e-mail sender (field `from`).
+- `APP_RDM_ADMIN_EMAIL_RECIPIENT`: modifies the e-mail recipient (field `to`).
+- `FILES_INTEGRITY_REPORT_SUBJECT`: modifies the  subject of the e-mail (field `subject`).
```

Comment:
We can remove the `FILES_INTEGRITY_REPORT_SUBJECT` from this section since it's more specific to one notification in particular (it's given as example below, so we don't lose informative documentation).

Reply:
Thanks, I've moved it!